homework-submission/week1/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './App.css';
+import Todo from './Todo'
+import Title from './Title';
+
+const todo = [
+  {
+    key: 1,
+    description: 'Get out of bed',
+    deadline: 'Wed Sep 13 2017'
+  },
+  {
+    key: 2,
+    description: 'Brush teeth',
+    deadline: 'Thu Sep 14 2017'
+  },
+  {
+    key: 3,
+    description: 'Eat breakfast',
+    deadline: 'Fri Sep 15 2017'
+  }
+]
+class App extends Component {
+  render() {
+    const todos = todo.map((item) => (<Todo key={item.key} description={item.description} deadline={item.deadline} />));
```

Comment:
you can use the internal key of the Map method instead of the item.id.
like so:
todo.map( ( item , key ) => ( < Todo key = { key } description = { item.description } deadline ={ item.deadline }  / > ) );